kcidb/oo/argparse.py

```diff
@@ -0,0 +1,49 @@
+""""KCIDB object-oriented (OO) data representation-management
+    command-line argument parsing
+ """
```

Comment:
Please remove the extra double quote at the beginning, the extra space before the closing `"""`, and format the docstring the same way as other docstrings are. I.e.:
```suggestion
"""
KCIDB object-oriented (OO) data representation-management
command-line argument parsing
"""
```